test/form-builder/components/FormConditionsModal.spec.js

```diff
@@ -121,7 +121,7 @@ describe('FormConditionsModal', () => {
     sinon.assert.calledOnce(closeSpy);
   });
 
-  it('should rerender ObsControlScriptEditorModal on change  of obs dropdown', () => {
+  it.skip('should rerender ObsControlScriptEditorModal on change  of obs dropdown', () => {
```

Comment:
@bsneha90 any specific reason for why its breaking. Can't we fix the test instead of skipping it.

Reply:
You are referring to an old commit, latest code does not skip it